documentation/sdk/docs/how_tos/native_tokens/destroy_foundry.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Destroy a Foundry
+sidebar_label: Destroy a Foundry
+description: 'How to destroy a foundry using wallet.rs.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- destroy
+- destroy foundry
+- foundry
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from '!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/native_tokens/destroy_foundry.ts';
+import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/native_tokens/destroy_foundry.py';
+import RustCode from '!!raw-loader!../../../../../sdk/examples/how_tos/native_tokens/destroy_foundry.rs';
+
+You can destroy a foundry by calling the `Account.burn(burn, options)` function. The function will
+destroy a foundry output as long as its circulating
+[native token](https://wiki.iota.org/shimmer/introduction/explanations/ledger/foundry) supply is zero. Any native tokens
+which were minted by other foundries will be sent to the controlling
+[alias](https://wiki.iota.org/shimmer/introduction/explanations/ledger/alias).
```

Comment:
> Any native tokens
which were minted by other foundries will be sent to the controlling

This isn't happening

Reply:
Who wrote it than? 😅 

```suggestion
[native token](https://wiki.iota.org/shimmer/introduction/explanations/ledger/foundry) supply is zero. 
```